src/MusicalScore/Graphical/VexFlow/VexFlowMeasure.ts

```diff
@@ -1135,9 +1172,29 @@ export class VexFlowMeasure extends GraphicalMeasure {
                     if (tupletStaveNotes.length > 1) {
                       const tuplet: Tuplet = tupletBuilder[0];
                       const notesOccupied: number = tuplet.Notes[0][0].NormalNotes;
-                      const bracketed: boolean = tuplet.Bracket ||
```

Comment:
here's where the main code changes outside of autobeams start (don't add bracket if not necessary)